### DISCUSSION TITLE
【作业提交】zkiap session 5

### DISCUSSION THREAD

**Discussion by Demian101** [zkp-co-learning/zkp-co-learn#47] [Show and tell]
【作业提交】zkiap session 5

**Comment by pochenai** [comment]
https://github.com/dajuguan/MIT-ZKIAP-2023/blob/main/Session_5_Exercises.md

**Reply by pochenai** [reply]
荐读Vitalik的这篇对Verkel Tree的直观讲解: https://vitalik.ca/general/2021/06/18/verkle.html

**Reply by Demian101** [reply]
COOL！

**Comment by Insomnia501** [comment]
https://github.com/Insomnia501/zkp-co-learning/blob/main/exercise/session5.md
